### DISCUSSION TITLE
please tell me your favorite sports!

### DISCUSSION THREAD

**Discussion by sakiho-k** [NAIST-SE/AttractingContributors#832] [2023 Attraction Contributions]
Could you please me what is your favorite sports? So please add your's lists of favorite sports to my README.md.

Please come to my fork:
https://github.com/sakiho-k/Please-tell-me-your-favorite-sports

**Comment by KKKKKXY** [comment]
I added my favorite sport into your list.(githubid: KKKKKXY)

**Comment by XelaAl** [comment]
Baseball. It's a perfect mix of strategy and athleticism. My friends and I often get together to watch games. We even called [Sports Gear Swag customer service](https://www.pissedconsumer.com/sports-gear-swag/RT-F.html) and ordered t-shirts with the prints of our favorite team. Do you also do your best not to miss matches?
